paasta_tools/cli/utils.py

```diff
@@ -124,6 +131,15 @@ def failure(msg, link):
     return "{} {} {}".format(x_mark(), msg, PaastaColors.blue(link))
 
 
+def info_message(msg: str) -> str:
+    """Format a paasta info message.
+
+    :param msg: a string
+    :return: a beautiful string
```

Comment:
probably fine to not add this since they don't buy us a ton (they're basically just repeating the type signature)